src/commands/moderation/ban.ts

```diff
@@ -1,120 +1,154 @@
-import { Colors, EmbedBuilder, EmbedField, Snowflake, TextChannel, User } from 'discord.js';
+import { Colors, TextChannel, User } from 'discord.js';
+import moment from 'moment';
 import { CommandDefinition } from '../../lib/command';
 import { Channels, CommandCategory } from '../../constants';
-import { makeEmbed } from '../../lib/embed';
+import { makeEmbed, makeLines } from '../../lib/embed';
 
-type UserLike = User | Snowflake
+const modLogEmbed = (formattedDate, moderator: User, user: User, reason: string) => makeEmbed({
+    color: Colors.Red,
+    author: {
+        name: `[BANNED] ${user.tag}`,
+        iconURL: user.displayAvatarURL(),
+    },
+    fields: [
+        {
+            name: 'Member',
+            value: user.toString(),
+        },
+        {
+            name: 'Moderator',
+            value: moderator.toString(),
+        },
+        {
+            name: 'Reason',
+            value: `\u200B${reason}`,
```

Comment:
Why zero space character?

Reply:
Can't remember, it was there before. I haven't tested with/without yet, but there was a reason for it at the time haha